modules/msg/src/main/MsgNotify.scala

```diff
@@ -71,13 +66,9 @@ final private class MsgNotify(
   private def doNotify(threadId: MsgThread.Id): Funit =
     colls.thread.byId[MsgThread](threadId.value) flatMap {
       _ ?? { thread =>
-        val msg  = thread.lastMsg
-        val dest = thread other msg.user
-        !thread.delBy(dest) ?? {
-          notifyApi addNotification Notification.make(
-            dest,
-            PrivateMessage(msg.user, shorten(msg.text, 40))
-          ) map (_ ?? lila.common.Bus.publish(MsgThread.Unread(thread), "msgUnread"))
```

Comment:
`lila.common.Bus.publish(MsgThread.Unread(thread), "msgUnread"))` is gone?

Reply:
Must have said something mean about Elon.